index.html

```diff
@@ -0,0 +1,1379 @@
+  <title>Newsweek</title>
+  <link rel="stylesheet" href="./styles.css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/2d6ecfbd70.js" crossorigin="anonymous"></script>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,300;0,400;0,700;1,300&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="header py-2 px-0 px-md-2">
+    <nav class="navbar navbar-expand navbar-light container-fluid">
+      <a href="#" class="navbar-brand d-lg-none">
+        <img class="logo" src="./img/logo-white.png" width="150" alt="logo">
+      </a>
+      <div class="collapse navbar-collapse">
+          <ul class="navbar-nav mr-auto d-block">
+            <li class="nav-item d-none d-lg-block">
+              <a href="#" class="text-white font-weight-bold white-small"><i class="fas fa-cloud-sun"></i> Fair  22°San José, CR</a>
+            </li>
+            <li class="nav-item d-none d-lg-block">
+              <a href="#" class="text-white white-small">Mon, Agus 03, 2020</a>
+            </li>
+          </ul>
+          <a href="#" class="navbar-brand d-none d-lg-block">
+            <img class="logo1" src="./img/logo-white.png" width="500" alt="logo">
+          </a>
+          <ul class="navbar-nav ml-auto align-items-end">
+              <li class="nav-item">
+                <a href="#" class="d-sm-block d-md-none text-white mr-2"><i class="fa fa-user"></i></a>
+              </li>
+              <li>
+                <a href="#" class="d-none d-md-block text-white mr-4 white-title">LOGIN</a>
+              </li>
+              <li>
+                  <a href="#" class="subscribe btn-primary">SUBSCRIBE <i class="fas fa-angle-right"></i></a>
+              </li>
+            </ul>
+            <button class="navbar-toggler d-block d-lg-none" type="button" data-toggle="collapse" 
+            data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+              <i class="fas fa-bars fa-2x"></i>
+            </button>
+          
+        </div>
+    </nav>
+  </header>  
+  <nav class="navbar navbar-expand-lg container-fluid bg-white w-100">  
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto buttons-menus font-weight-bold">
+          <li class="nav-item my-2">
+            <a href="#" class="border-right px-2 py-1 nav-link">U.S</a>
+          </li>
+          <li class="nav-item my-2">
+              <a href="#" class="border-right px-2 py-1 nav-link">Word</a>
+          </li>
+          <li class="nav-item my-2">
+              <a href="#" class="border-right px-2 py-1 nav-link">Business</a>
+          </li>
```

Comment:
- [ ] This section needs more width.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89329854-cac26580-d665-11ea-989c-f8e64fd92c97.png) |![image](https://user-images.githubusercontent.com/61479648/89329902-dca40880-d665-11ea-9b82-092b7eccfbd3.png) |